README.md

```diff
@@ -23,7 +23,7 @@ Further documentation
 - Developer wiki: http://dev.minetest.net/
 - Forum: http://forum.minetest.net/
 - Github: https://github.com/minetest/minetest/
-- doc/ directory of source distribution
+- [doc/](https://github.com/minetest/minetest/tree/master/docs/) directory of source distribution
```

Comment:
1) incorrect URL
2) a relative url should work here: `[doc](doc/)`

Reply:
Done, thank you!